### ISSUE TITLE
Error Daemon: Eggs list panel

### ISSUE THREAD

**Issue by patricknasci** [pterodactyl/panel#1373]
**Background (please complete the following information):**
* Panel or Daemon: Daemon
* Version of Panel/Daemon: v0.7.10/v0.6.7 
* Server's OS: Centos 7
* Your Computer's OS & Browser: Chrome 

**Describe the bug**
Error while attempting to fetch list of Eggs from the panel.
```
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"+ ------------------------------------ +","time":"2018-11-08T02:18:33.062Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"| Running Pterodactyl Daemon v0.0.0-canary    |","time":"2018-11-08T02:18:33.063Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"|        https://pterodactyl.io        |","time":"2018-11-08T02:18:33.063Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"|  Copyright 2015 - 2018 Dane Everitt  |","time":"2018-11-08T02:18:33.063Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"+ ------------------------------------ +","time":"2018-11-08T02:18:33.063Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Loading modules, this could take a few seconds.","time":"2018-11-08T02:18:33.063Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Modules loaded, starting Pterodactyl Daemon...","time":"2018-11-08T02:18:33.495Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":60,"err":{"message":"ENOENT: no such file or directory, open 'logs/wings.log'","name":"Error","stack":"Error: ENOENT: no such file or directory, open 'logs/wings.log'","code":"ENOENT"},"msg":"A fatal error occured during an operation.","time":"2018-11-08T02:18:33.519Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":60,"err":{"message":"ENOENT: no such file or directory, open 'logs/request.log'","name":"Error","stack":"Error: ENOENT: no such file or directory, open 'logs/request.log'","code":"ENOENT"},"msg":"A fatal error occured during an operation.","time":"2018-11-08T02:18:33.519Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Creating keypair to use for SFTP connections.","time":"2018-11-08T02:18:33.558Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Configuring user pterodactyl (id: 997) as the owner of all server files.","time":"2018-11-08T02:18:35.016Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Configuring timezone file location...","time":"2018-11-08T02:18:35.018Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Checking container networking environment...","time":"2018-11-08T02:18:35.020Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Found network interface for daemon: pterodactyl_nw","time":"2018-11-08T02:18:35.030Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Ensuring correct network interface for containers...","time":"2018-11-08T02:18:35.030Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Networking gateway detected as 172.18.0.1 for interface: pterodactyl0.","time":"2018-11-08T02:18:35.038Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":30,"msg":"Contacting panel to retrieve a list of currrent Eggs available to the node.","time":"2018-11-08T02:18:35.039Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":60,"err":{"message":"Error while attempting to fetch list of Eggs from the panel.","name":"Error","stack":"Error: Error while attempting to fetch list of Eggs from the panel.\n    at Request [as _callback] (/srv/daemon/src/controllers/service.js:95:31)\n    at Request.self.callback (/srv/daemon/node_modules/request/request.js:185:22)\n    at emitTwo (events.js:126:13)\n    at Request.emit (events.js:214:7)\n    at Request.<anonymous> (/srv/daemon/node_modules/request/request.js:1157:10)\n    at emitOne (events.js:116:13)\n    at Request.emit (events.js:211:7)\n    at IncomingMessage.<anonymous> (/srv/daemon/node_modules/request/request.js:1079:12)\n    at Object.onceWrapper (events.js:313:30)\n    at emitNone (events.js:111:20)"},"additional":{"responseCode":500,"requestURL":"https://gcp.cenathost.com/api/remote/eggs"},"msg":"A fatal error caused the daemon to abort the startup.","time":"2018-11-08T02:18:35.215Z","v":0}
{"name":"wings","hostname":"s1-games","pid":5274,"level":50,"msg":"You should forcibly quit this process (CTRL+C) and attempt to fix the issue.","time":"2018-11-08T02:18:35.215Z","v":0}
```

Panel: `php -v` (the php version in use).
```
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.11, Copyright (c) 1999-2018, by Zend Technologies
```
Daemon: `uname -a` and `docker info` (your kernel version and information regarding docker)

```
Linux s1-games 3.10.0-862.14.4.el7.x86_64 #1 SMP Wed Sep 26 15:12:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
```
```
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 18.09.0
Storage Driver: overlay2
 Backing Filesystem: xfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file local logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: c4446665cb9c30056f4998ed953e6d4ff22c7c39
runc version: 4fc53a81fb7c994640722ac585fa9ca548971871
init version: fec3683
Security Options:
 seccomp
  Profile: default
Kernel Version: 3.10.0-862.14.4.el7.x86_64
Operating System: CentOS Linux 7 (Core)
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 1.656GiB
Name: s1-games
ID: GKCS:73CH:CNW5:C7FE:TAK7:M4DP:DKUA:OVSU:B5VM:NPI2:YUWA:5KYI
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine

```

**Comment by parkervcp** [comment]
You should probably run the daemon in debug mode by setting level to `debug` from `info`

**Comment by FragnetNetworks** [comment]
Had the exact same issue with one of our daemons running on v0.6.7
Seems to be related to #1369 - temporary fix is here #1372

**Comment by DaneEveritt** [comment]
You have a bug related to your panel code, its returning a 500 error. Use the Discord channel for additional help getting the logs.

Thanks.
